### PR TITLE
Feature/init and docs enhancements

### PR DESCRIPTION
**Pull request by nbesimi**

### CODE REVIEW

File: cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java [hunk 1]
```diff
@@ -324,6 +323,54 @@ private void init(@CommandLine.Parameters(index = "0", description = "Project na
         if (!projectName.isEmpty()) {
             log.info("In order to start using the newly created project please change your working directory.");
         }
+
+
+        Path driversPath = Path.of(DEFAULT_DRIVERS_YAML);
+
+        DriverHelper.printDrivers(driversPath);
+        System.out.println("Which is the source DB you plan to use (default: skip)?");
```
Comment (ailegion) [inline cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java:331]: /nit
Which source database do you plan to use? (default: skip)

File: cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java [hunk 2]
```diff
@@ -324,6 +323,54 @@ private void init(@CommandLine.Parameters(index = "0", description = "Project na
...
+        String sourceDB = new BufferedReader(new InputStreamReader(System.in)).readLine().trim();
+        if (sourceDB.isEmpty()) {
+            sourceDB = "skip";
+        }
+        handleDriverDownload(sourceDB, driversPath, "source");
+
+        DriverHelper.printDrivers(driversPath);
+        System.out.println("Which is the target DB you plan to use (default: skip)?");
```
Comment (ailegion) [inline cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java:339]: /nit
Which target database do you plan to use? (default: skip)

File: cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java [hunk 3]
```diff
@@ -324,6 +323,54 @@ private void init(@CommandLine.Parameters(index = "0", description = "Project na
...
+            log.info("Skipped downloading the {} DB driver.", dbType);
+            return;
+        }
+
+        Integer driverId = null;
+
+        try {
+            driverId = Integer.parseInt(dbChoice);
+        } catch (NumberFormatException e) {
+            System.out.println("Invalid index. Please provide a valid index.");
```
Comment (ailegion) [inline cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java:358]: /nit
Presuming this for when user is provided numbered option to select from:
Invalid choice. Please select a valid option.

File: cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java [hunk 4]
```diff
@@ -324,6 +323,54 @@ private void init(@CommandLine.Parameters(index = "0", description = "Project na
...
+        }
+
+        List<DriverInfo> drivers = DriverHelper.getDrivers(driversPath);
+        if (drivers.isEmpty()) {
+            System.out.println("No drivers found in the specified YAML file.");
+            return;
+        }
+
+        if (driverId < 1 || driverId > drivers.size()) {
+            System.out.println("Invalid index. Please provide a valid index.");
```
Comment (ailegion) [inline cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java:369]: /nit
Same as above
Invalid choice. Please select a valid option.

**Review by jetroni — APPROVED** [review]

**Pull request merged**